server/src/utils/validate.ts

```diff
@@ -0,0 +1,67 @@
+import { firstUC } from ".";
+import { ValidationBody } from "../types";
+
+function validateEmail(email) {
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(String(email).toLowerCase());
+}
+
+export default (body: Array<ValidationBody>): { errors: object; values?: object } => {
+  const errors = {};
+  let values = {};
+
+  if (!body.length) return { errors };
+
+  if (body.length) {
```

Comment:
do you need to put this condition here? It the above fails, following block will considered.